ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -26,6 +26,11 @@ def diff(a, n=1, axis=-1, prepend=None, append=None):
     return ivy.diff(a, n=n, axis=axis, prepend=prepend, append=append, out=None)
 
 
+@to_ivy_arrays_and_back
+def ediff1d(ary, to_end=None, to_begin=None):
+    return ivy.ediff1d(ary, to_end=to_end, to_begin=to_begin, out=None)
```

Comment:
I mean you should write compositional implementation using existing ivy functions here.

Reply:
I have used `ivy.diff` to implement this function. Please let me know if there is a problem